Engineering/CodeReviews/Java.md

```diff
@@ -0,0 +1,36 @@
+# Java Code Reviews
+
+Please refer to the general [CSE](../../CSE.md) [code review guidance](../CodeReviews.md) as well. Provided here is a check list of Java specific items to help you with your code reviews.
+
+## Code Style
+
+Choose a code style (or coding standard) for your project, stick to it and enforce it. There are several accepted ones to choose from. Good places to start are:
+
+* [Google Java Style Guide](https://checkstyle.org/styleguides/google-java-style-20170228.html)
+* [Oracle's Code Conventions for the Java Programming Language](https://www.oracle.com/technetwork/java/javase/documentation/codeconvtoc-136057.html)
+
+To enforce a coding standard, you can lean on a number of tools for this purpose:
+
+* [checkstyle](http://checkstyle.sourceforge.net/) - "_a development tool to help programmers write Java code that adheres to a coding standard. It automates the process of checking Java code to spare humans of this boring (but important) task._"

```

Comment:
Another very common static analysis tool is `findbugs` which is apparently now called `spotbugs`. I suggest adding it here.
https://github.com/findbugsproject/findbugs